src/renderer/pages/project/script_editor/text_editor.vue

```
@@ -56,12 +56,13 @@
   beat: MulmoBeat;
   audioFile?: string;
   projectId: string;
+  mulmoMultiLingual: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)